README.md

```diff
@@ -2,89 +2,194 @@
-  - Run the hardhat `fund` task:
+# Install the dependencies - Note your Node version 'Requirements' 
+yarn install
 ```
-cd ../contracts
-HARDHAT_NETWORK=localhost npx hardhat fund --amount 10000
+
+#### Standalone Mode
+```bash
+# Run the local hardhat node
+yarn run node
 ```
 
+#### Forked Mode
 
-### Running Smoke Tests
+You will need a provider to run in forked mode. Check out [Infura](https://infura.io/) or similar. You can do this on mainnet or a testnet.
 
-Smoke tests can be ran in 2 modes: 
-- Run `scripts/test/smokeTest.sh` to launch interactive mode. All the "before contract changes" parts of tests
-  will execute and wait for the user to manually using a console performs contract changes. Once those are done
-  hit "Enter" in the smoke test console and the second part of the tests shall be ran that validate that contract
-  changes haven't broken basic functionality.
-- Run `scripts/test/smokeTest.sh --deployid [numeric_id_of_deploy]` will run smoke tests against a specific
-  deployid. Validating that that deploy didn't break basic functionality.
+```bash
+# export your provider(Infura, Truffle Teams, Alchemy, etc)
+export PROVIDER_URL=<provider url>
 
-### Runnning on Mainnet fork
+# optional - set the block number you want to fork at - 6 confirmations are suggested
+export BLOCK_NUMBER=<block number>
 
-This repository also supports running a local node via the Hardhat EVM fork implementation.
+# Run the local hardhat node in forked mode
+yarn run node:fork
+```
 
-Set PROVIDER_URL to point to Mainnet.
+### Minting Stablecoins on the Local Hardhat Node
+You will be needing stablecoins such as `USDT`, `USDC`, `DAI`, etc to mint the `OUSD` coin for usage in the dApp. You can do this in several ways:
+- run a hardhat task
+- visit http://localhost:3000/dashboard
+<br/><br/>
 
-`export PROVIDER_URL="https://eth-mainnet.alchemyapi.io/v2/...`
+#### Hardhat Task
+Open a new terminal with your local hardhat node still running.
+```bash
+cd contracts
+```
 
-Setting BLOCK_NUMBER in your environment before running the fork will aid performance due to better caching. It is recommended to set it to something that hast at least 6 confirmations.
+##### Standalone Mode
+```bash
+# Mint 1000 of each supported stablecoin to each account defined in the mnemonic
+npx hardhat fund --amount 1000 --network localhost
+```
 
-`export BLOCK_NUMBER=...`
+##### Forked Mode
+```bash
+# Mint 1000 of each supported stablecoin to each account defined in the mnemonic
+HARDHAT_NETWORK=localhost npx hardhat fund --amount 1000
+FORK=true npx hardhat fund --amount 1000 --network localhost
+```
+Note:
+You may need to remove [this line](https://github.com/nomiclabs/hardhat/blob/fc50a94a688ed5007a429857b808aae76441095c/packages/hardhat-core/src/internal/core/providers/http.ts#L119) from `node_modules/hardhat` if you experience timeouts running on the fork. Timeouts occur because of how long it takes to retrieve the blockchain state from the provider.
```

Comment:
Have you experieced any timeouts when developing? This PR should fix the timeout issue: https://github.com/OriginProtocol/origin-dollar/pull/552 . Though it hasn't been tested through all the various hardhat tasks

Reply:
I just left it in there - I havn't done a lot with the mainnet fork. I can remove this if you think @sparrowDom

Reply:
I removed that blurb @sparrowDom

Reply:
Thanks a bunch!